.test-infra/dataproc/init-actions/flink.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This file was copied from https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/blob/master/flink/flink.sh 

Is it ok to copy the file this way? What license should I put there? Should I change it?

Reply:
Apache license looks like it's okay.